Solution4.py

```diff
@@ -0,0 +1,21 @@
+# Question 4 Solution
+import csv
+header=['sid','name','salary']
+rows=[[1,'krishna',4600],[2,'om',4000],[2,'ram',4000],[3,'radha',4600],[5,'sai',4200]]
```

Comment:
can we take input from user? if yes...please do it

Reply:
yes sir, i will make the required changes